i2c/mcp9808_i2c/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Attaching a MCP9808 digital temperature sensor via I2C
+
+This example code shows how to interface the Raspberry Pi Pico to the MCP9808 digital temperature sensor board. 

```

Comment:
Is there a publicly-available datasheet available online somewhere that we can link to, for the benefit of people who might want to tweak this example to do other things?

Reply:
Yup! Will pop that on